guides/micronaut-data-access-mybatis/java/src/main/java/example/micronaut/ListingArguments.java

```diff
@@ -28,7 +29,15 @@ public class ListingArguments {
     @Nullable
     private String order;
 
-    public ListingArguments() {
+    private ListingArguments() {
```

Comment:
can we remove this private constructor?

Reply:
The private constructor is used by the builder